### ISSUE TITLE
ClassCastException on message receive (I think)

### ISSUE THREAD

**Issue by xXAndrew28Xx** [DV8FromTheWorld/JDA#144]
Happens right when I start my bot, maybe it happens when it receives a message. I have one message listener. Full source of my bot is on https://github.com/xXAndrew28Xx/SMORE-BOT
```
[18:06:38] [Fatal] [JDASocket]: java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
        at net.dv8tion.jda.handle.EntityBuilder.createGuildFirstPass(EntityBuilder.java:144)
        at net.dv8tion.jda.handle.GuildJoinHandler.handleInternally(GuildJoinHandler.java:38)
        at net.dv8tion.jda.handle.SocketHandler.handle(SocketHandler.java:38)
        at net.dv8tion.jda.requests.WebSocketClient.handleEvent(WebSocketClient.java:629)
        at net.dv8tion.jda.requests.WebSocketClient.onTextMessage(WebSocketClient.java:324)
        at net.dv8tion.jda.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:705)
        at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:368)
        at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:243)
        at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:937)
        at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:696)
        at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:102)
        at com.neovisionaries.ws.client.ReadingThread.run(ReadingThread.java:61)
```

**Comment by MinnDevelopment** [comment]
This was fixed in the most recent commit.

**Comment by DV8FromTheWorld** [comment]
This has nothing to do with message receiving, it has everything to do with stuff related to Game status for presences. This has been fixed as of feb0aa2cf252ea69327a94d70abd99323867254a. Please update.
